packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -8,7 +8,7 @@ export const comments: { readonly [key in TokenName]: string | null } = {
     applicationBackgroundColor: 'Primary background color for the application',
     dividerBackgroundColor: 'Divider background color',
     headerBackgroundColor: 'Background color for application headers',
-    sectionBackgroundColor: 'Background color for dialogs and menus',
+    sectionBackgroundColor: 'Background color for menus and dropdown lists',
```

Comment:
I think this comment is wrong, not the color associated with it. We do have a concept of sections and section colors, and the colors of Black15 and Black80 are correct for it (no color is in the design spec for the Color theme yet). Therefore, I think we should:
- keep this token but update the comment to be accurate
- if deemed necessary, add a new token named something like `popupBackgroundColor`

If you change the meaning of this existing token, it'll break SLE because the `$ni-nimble-section-background-color` token is used in a number of different places for panels that are displayed above grid (e.g. the summary panel and the details panel).

Reply:
Thanks for pointing this out. I've reverted everything, updated the comment, and took the approach that was used on the menu.